tests/integration/linux_bridge_test.yaml

```diff
@@ -0,0 +1,248 @@
+description: Configure eth2 with the controller linux-br0
+interfaces:
+- name: eth2
+  type: ethernet
+  state: up
+  controller: linux-br0
+description: Create the linux bridge linux-br0 with the ethernet port eth1 and eth2. For both ethernet port eth1 and eth2, do not allow traffic to be sent back to the same port on which it was received, configure the stp path cost to 100, configure the stp priority to 32
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  bridge:
+    port:
+    - name: eth1
+      stp-hairpin-mode: false
+      stp-path-cost: 100
+      stp-priority: 32
+    - name: eth2
+      stp-hairpin-mode: false
+      stp-path-cost: 100
+      stp-priority: 32
+description: Create the bond interface testbond0 with the controller linux-br0, the bonding mode balance-rr, and ethernet port eth1
+interfaces:
+- name: testbond0
+  type: bond
+  state: up
+  controller: linux-br0
+  link-aggregation:
+    mode: balance-rr
+    port:
+    - eth1
+description: Configure the linux bridge linux-br0 with the group address 01:80:C2:00:00:04
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  bridge:
+    options:
+      group-addr: 01:80:C2:00:00:04
+description: Configure the linux bridge linux-br0 with multicast-snooping set to true
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  bridge:
+    options:
+      multicast-snooping: true
+description: Set the controller of ethernet device eth1 to empty
+interfaces:
+- name: eth1
+  type: ethernet
+  state: up
+  controller: ''
+description: bring up the linux bridge linux-br0
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+description: Configure the linux bridge linux-br0 with port eth2 attached, do not allow traffic to be sent back to the same port on which it was received, set the stp path cost to 100, set the stp priority to 32
```

Comment:
We already explained the hairpin-mode in previous test, I think it is better just say `disable stp hairpin-mode` here as diversity.